### DISCUSSION TITLE
BREAKING CHANGE: Considering to remove the possibillity of manually adding `FluentOption`s to list components in a next version

### DISCUSSION THREAD

**Discussion by vnbaaij** [microsoft/fluentui-blazor#1366] [Ideas]
I think it would be better if AutoComplete, Listbox, Combobox and Select would only get their options specified through the `Items` parameter (`IEnumerable<TOption>`). 

By having the option of people specifying the items in the list manually (with `FluentOption` components) a disconnect exists between what the DOM knows about a list component and what Blazor knows about the list component. I think that is not desirable as, for example, things like keyboard selection of manually added items is impossible to do.

Before we decide to make such a (breaking) change, I would like to learn more from you. 
- Are you using manual options in the list components? 
- Are you running into issues with it? 
- Would removing that option be a problem for you and why?

**Comment by tomokusaba** [comment]
Can I mark it as obsolete once?
We need time to address this issue.

**Reply by vnbaaij** [reply]
It is not for **the** next version, but for **a** next version. If we decide to do it, the version it will be in is tbd.
